ttnn/ttnn/operations/math.py

```diff
@@ -53,11 +53,15 @@ def _golden_function(input_tensor: ttnn.Tensor, **_):
         return torch_function(input_tensor)
 
     def _math_op_validate_input_tensors(operation_name, input_tensor, *args, **kwargs):
+        if operation_name == "ttnn.erfinv":
+            supported_dtypes = (ttnn.bfloat16,)
+        else:
+            supported_dtypes = (ttnn.bfloat16, ttnn.bfloat8_b)
         ttnn.validate_input_tensor(
```

Comment:
If the validation on the dtypes needs to be different we need a completely different operation here.   The reason is because this will not port well to C++ with how we are currently doing it.